setup/freebsd/ansible-vars.yaml

```diff
@@ -4,8 +4,9 @@ init_script_path: /usr/local/etc/rc.d/jenkins
 ssh_users:
   - rvagg
   - jbergstroem
+  - joaocgreis
```

Comment:
Is this intentional? Seems like the kind of thing that should get a mention in the commit message if so.